### ISSUE TITLE
vignettes not found

### ISSUE THREAD

**Issue by COrphanides** [AustralianAntarcticDivision/ZooScatR#1]
Hello, 

I am new to Github, so if this is not the right place for this question, let me know. I am exploring using your package for DWBA calculations. I was able to install it and get things to run, except that it has not been able to find the vignettes, which would be helpful for learning how to use it best.

> vignette(package="ZooScat")
no vignettes found
> browseVignettes("ZooScatR")
No vignettes found by browseVignettes("ZooScatR")

I worked through the installation a few times to make sure all the appropriate packages were installed and of the right version. From some of your help information, it looks like the vignettes should be in a doc folder (e.g., ../doc/DwbaCommand.html), which doesn't appear to be installed on my system.

The installation log looks good except for a warning at the end, here are the last several lines:

*** copying figures
** building package indices
** installing vignettes
** testing if installed package can be loaded
*** arch - i386
Warning: replacing previous import 'shiny::runExample' by 'shinyjs::runExample' when loading 'ZooScatR'
*** arch - x64
Warning: replacing previous import 'shiny::runExample' by 'shinyjs::runExample' when loading 'ZooScatR'
* DONE (ZooScatR)
In R CMD INSTALL
Warning messages:
1: In untar2(tarfile, files, list, exdir) :
  skipping pax global extended headers
2: In untar2(tarfile, files, list, exdir) :
  skipping pax global extended headers

Thanks for putting this package together, I am excited for its possibilities.

-Chris

**Comment by SvenGastauer** [comment]
Hi Chris,

I suspect this has to do with your version of devtools, as they recently changed the syntax.

Could you tell me what you are getting for:

```packageVersion("devtools")```

If it says > 2.0, try:

```devtools::install_github("AustralianAntarcticDivision/ZooScatR", build_opts = c("--no-resave-data", "--no-manual"))```

otherwise for <2.0 try: 

```devtools::install_github("AustralianAntarcticDivision/ZooScatR", build_vignettes = TRUE, force_deps=TRUE)```

If the vignettes are being installed, the installation will take a while and during the process, a pop-up with a progress bar, showing you the progress of the simulation vignette show become visible.

In case it doesn't work, a temproary solution could be accessing the vignettes through [gitHub](https://github.com/AustralianAntarcticDivision/ZooScatR/tree/master/doc)

Hope this helps!

-Sven

**Comment by SvenGastauer** [comment]
Sorry forgot to mention...The warnings should be fine...They are caused by the shinyjs package but should not cause any headaches.

**Comment by COrphanides** [comment]
Hello. Thanks for your reply. I had devtools 2.0.1.  I removed the old devtools and ran your code for the installation. I got a Windows warning saying about something being blocked by the Windows firewall when the vignettes were being installed.  Running this function vignette(package="ZooScat") still results in finding no vignettes, but now browseVignettes("ZooScatR") does allow me to get to the vignettes with my browser. So, thanks for that. I may have one more question, but I'll hold off until I explore the package further.
